feature/xhr/aggregate/index.js

```diff
@@ -168,3 +178,8 @@ function onEventsHarvestFinished(result) {
     sentAjaxEvents = []
   }
 }
+
+function exceedsSizeLimit(payload) {
+  return new Blob([payload]).size > loader.maxBytes
```

Comment:
Blob is not supported on all browsers - we should have a feature detection around this. There are not many browsers that do not support it, but we do not want to cause errors anywhere. Also, I think it's ok drop the data for those old browsers when the payload size is too large (if it is not easy to do something else).